app/models/document.rb

```diff
@@ -1,7 +1,7 @@
 class Document
   attr_reader :title, :public_timestamp, :is_historic, :government_name,
               :content_purpose_supergroup, :document_type, :organisations,
-              :release_timestamp, :es_score
+              :release_timestamp, :es_score, :description
```

Comment:
As @steventux mentioned in another comment, I don't think you need to expose `description` as `summary` is already available.

Reply:
summary always comes back as null, so have to get the description from the document and add it to top result

Reply:
That's because `show_summaries` isn't set in the content_item. However, I think that if we set that to true, the summary will show for every list item.

The only concern I have is that `summary` returns a truncated description. We might want to do the same thing.